library/catalog/views.py

```diff
@@ -309,3 +312,41 @@ def collections_page(request):
 def authors_page(request):
     authors = Author.objects.filter(status=Status.ACTIVE).order_by('name')
     return render(request, 'authors.html', {'authors': authors})
+
+
+def reset_password(request):
+    if request.method == 'POST':
+        email = request.POST.get('email')
+        user = MongoUser.objects.filter(email=email).first()
+        if user:
+            new_password = ''
+            numbers_list = [str(i) for i in range(21)]
```

Comment:
MAX_PASSWORD_NUM = 21
numbers_list = [str(i) for i in range(MAX_PASSWORD_NUM )]

Don't forget to put constants to the top of the module, after the imports